interval/interval_test.go

```diff
@@ -0,0 +1,82 @@
+package interval
+
+import (
+	"testing"
+	"time"
+)
+
+func TestRandomInstant(t *testing.T) {
+	tests := []struct {
+		desc     string
+		iv       *Interval
```

Comment:
eek! initialization vector.  Not sure what else you can call this.  Sometimes I use 'in' indicating 'input'.

Reply:
Changed :)